### PR TITLE
OSDOCS-5412: fixing broken links in install guide

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-5412

Link to docs preview:
https://56319--docspreview.netlify.app/microshift/latest/microshift_install/microshift-install-rpm.html#installing-microshift-from-rpm-package_microshift-install-rpm
scroll up a little from
https://56319--docspreview.netlify.app/microshift/latest/microshift_install/microshift-embed-in-rpm-ostree.html#accessing-microshift-cluster_microshift-embed-in-rpm-ostree

QE review:
N/A internal docs bug

### CODE REVIEW

File: microshift_install/microshift-embed-in-rpm-ostree.adoc [hunk 1]
```diff
@@ -51,9 +51,9 @@ include::modules/microshift-provisioning-ostree.adoc[leveloffset=+1]
 .Additional resources
 
 * link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/composing_installing_and_managing_rhel_for_edge_images/index[{op-system-ostree} documentation].
-* xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[System requirements for installing {product-title}].
+* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-system-requirements_microshift-install-rpm[System requirements for installing MicroShift].
```
Comment (JoeAldinger) [inline microshift_install/microshift-embed-in-rpm-ostree.adoc:54]: So much better. Awesome!

File: microshift_install/microshift-install-rpm.adoc [hunk 2]
```diff
@@ -16,27 +16,27 @@ Red Hat does not support an update path from Developer Preview and Technology Pr
...
+[role="_additional-resources"]
+[id="_additional-resources_microshift-install-rpm"]
 .Additional resources
 
-* xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[System Requirements for Installing MicroShift] have been met.
-
 * Download the link:https://console.redhat.com/openshift/install/pull-secret[pull secret] from the Red Hat Hybrid Cloud Console.
 
-* xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-using-config-tools[Configuring MicroShift].
+* link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/configuring/microshift-using-config-tools[Configuring MicroShift].
```
Comment (JoeAldinger) [inline microshift_install/microshift-install-rpm.adoc:25]: Not sure if you want to switch this one or not to an xref.

Comment (ShaunaDiaz) [reply microshift_install/microshift-install-rpm.adoc]: Pushing...

**Review by JoeAldinger — COMMENTED** [review]
A few small suggestions. 
/remove-label peer-review-in-progress
/label peer-review-done

**Pull request merged**